packages/grid/_modules_/grid/components/panel/GridPanel.tsx

```diff
@@ -2,6 +2,7 @@ import * as React from 'react';
 import clsx from 'clsx';
 import { Theme } from '@material-ui/core/styles';
 import { makeStyles } from '@material-ui/styles';
+import { generateUtilityClasses } from '@material-ui/unstyled';
```

Comment:
unstyled is not a dependency of the packages, it would break people's build. Trying to open a codesandbox from the docs preview. 

Would this approach work instead? https://github.com/mui-org/material-ui-x/blob/a4df486a95b72847224b71291fcdc3211feea3d1/packages/grid/_modules_/grid/utils/material-ui-utils.ts#L61

Reply:
Hmm I noticed this utils file but I found easier to import from @material-ui/unstyled. I didn't realize that it would break the build in v4. I'll add the `generateUtilityClasses` to it.

Reply:
Actually, I'm updating #1902 so we don't forget about this